modules/gis/src/lib/flat-geojson-to-binary.ts

```diff
@@ -154,7 +157,7 @@ function fillArrays(
         ? new Uint32Array(polygonRingsCount + 1)
         : new Uint16Array(polygonRingsCount + 1),
     positions: new PositionDataType(polygonPositionsCount * coordLength),
-    triangles: [],
+    ...(triangulate && {triangles: []}),
```

Comment:
Nit: I find spreads like this hard to read. Not really worth it to save a lint. I'd probably use a separate if statement to add the field.